cmd/kubectl-gadget/audit/audit.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019-2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package audit
+
+import (
+	"errors"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+
+	"github.com/spf13/cobra"
+)
+
+// All the gadgets within this package use this global variable, so let's
+// declare it here.
+var params utils.CommonFlags
+
+var AuditCmd = &cobra.Command{
+	Use:   "audit",
+	Short: "Audit a subsystem",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return errors.New("you should not call this without any sub-command")
+	},
```

Comment:
IMO, it is enough to show the usage. BTW, by removing this, the flag line in the usage is removed:

```
$ ./kubectl-gadget audit
Audit a subsystem

Usage:
  kubectl-gadget audit [command]

Available Commands:
  seccomp     Trace syscalls that seccomp sent to the audit log
``

Reply:
See [this](https://github.com/kinvolk/inspektor-gadget/pull/500#discussion_r822524046) comment for context.